src/app/app.component.html

```diff
@@ -0,0 +1,40 @@
+<nav>
```

Comment:
@hitpopdimestop I think it is more convenient to name this file as `app.template.html` like in home component